src/Auditor.jl

```diff
@@ -209,6 +209,33 @@ function audit(prefix::Prefix, src_name::AbstractString = "";
         rm(f; force=true)
     end
 
+    # Make sure that `prefix` in pkg-config files *.pc points to the right
+    # directory.
+    pc_files = collect_files(prefix, endswith(".pc"))
+    for fs in pc_files
+        # Make sure the file still exists on disk
+        if !isfile(fs)
+            continue
+        end
+
+        fs = open(file, "r")
+
+        # We want to replace every instance of `prefix=...` with
+        # `prefix=${pcfiledir}/../..`
+        content = ""
+        str = readline(fs, keep = true)
+        while !isempty(str)
+            tmp = replace(str, r"^prefix=.*$" => s"prefix=${pcfiledir}/../..")
```

Comment:
This looks *potentially* wrong, you can't rewrite an arbitrary path like, you should rewrite it relative to the prefix, for example use [`relpath`](https://docs.julialang.org/en/v1/base/file/#Base.Filesystem.relpath)

Reply:
I believe this is correct. Or do you mean that the path should be semi-hardcoded, and not use `${pcfiledir}`?

Reply:
I mean that you're assuming that the pkgconfig files will ***always*** be in the same directory, and same relative path to the prefix, which is probably true in 99% of the cases, but I don't want to hardcode this assumption.

Reply:
Wouldn't that be a bug in that case? That's the default for both Unix-type and Windows systems, no?

Reply:
Pkgconfing files can be in multiple directories, not just `${prefix}/lib/pkgconfig`: they could also be `${prefix}/lib64/pkgconfig` or `${prefix}/share/pkgconfig`, and that's totally fine and normal.  The prefix _happens_ to have the same relative path compared to all these directories, but it isn't necessarily the case. Why hardcoding assumptions and run into potential troubles when things can be dealt with automatically?

Reply:
I have implemented this.